### ISSUE TITLE
Cast Error on trying to create page

### ISSUE THREAD

**Issue by RenatoCesarF** [joseph-grabinger/flutter_to_pdf#75]
Hi, I''m trying to export a complex page, but always receive some of those errors:
- type 'Null' is not a subtype of type 'AlignmentDirectional' in type cast
- type 'EdgeInsetsDirectional' is not a subtype of type 'EdgeInsets' in type cast

My flutter version:
Flutter 3.19.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 68bfaea224 (4 months ago) • 2024-03-20 15:36:31 -0700
Engine • revision a5c24f538d
Tools • Dart 3.3.2 • DevTools 2.31.1

I tried export just a simple Widget and it works, but not all my page or a more complex one.

Don't know if its the package version, the flutter version or some configuration.

Follows more of my code:

```dart
    ExportFrame(
                exportDelegate: controller.exporter,
                frameId: 'analytics',
                child: Widget(...))
```

```dart
  @action
  Future<void> export(BuildContext context) async {
    try {
      final pdf = await exporter.exportToPdfDocument('analytics');
      await saveFile(pdf, 'teste');
      SuccessMessage.show(
          context, "Feedback enviado com sucesso! Muito Obrigado");
    } catch (e, stackTrace) {
      print(stackTrace);
      print(e);
      ErrorMessage.show(context, e.toString());
    }
  }
```

```dart
  ExportDelegate exporter = ExportDelegate(
      options: const ExportOptions(
    checkboxOptions: CheckboxOptions.none(),
    pageFormatOptions: PageFormatOptions.a4(),
  ));
```

**Comment by sousa-p** [comment]
I had the same error, I'm trying to find a solution, if I get any results, I'll let you know!

PR: [https://github.com/joseph-grabinger/flutter_to_pdf/pull/76](url)

**Comment by sousa-p** [comment]
I had the same error, I tried this solution:

[https://github.com/joseph-grabinger/flutter_to_pdf/pull/77](url)

**Comment by joseph-grabinger** [comment]
> I had the same error, I tried this solution:
> 
> PR: [https://github.com/joseph-grabinger/flutter_to_pdf/pull/77](url)

Will be closed by #77
